big5/workload.json

```diff
@@ -35,7 +35,7 @@
             "compressed-bytes": 53220934846,
             "uncompressed-bytes": 943679382267
           }
-        {% elif corpus_size == 1000-full %}
+        {% elif corpus_size == "1000-full" %}
           {
             "source-file": "documents-1000-full.json.bz2",
```

Comment:
Do we also need to include this in files.txt?

Reply:
https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/big5/files.txt